terraform/cert-manager.tf

```diff
@@ -28,7 +32,7 @@ data "aws_iam_policy_document" "cert_manager" {
       "route53:ListResourceRecordSets"
     ]
     resources = [
-      "arn:aws:route53:::hostedzone/Z06469063HC60S7SHF706"
+      "${data.aws_route53_zone.this.id}"
```

Comment:
This won't work `id` just returns `Z06469063HC60S7SHF706` you need to do `"arn:aws:route53:::hostedzone/${data.aws_route53_zone.this.id}"`

Reply:
ok